packages/dashboard/src/components/Card/examples/MultiCountExample.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from 'react';
+import { ICardProps, CardSize, Priority, CardContentType } from '../Card.types';
+import { Card } from '../Card';
+import { IAction } from '../ActionBar/ActionBar.types';
+import { AnnotationType } from '@uifabric/dashboard';
+import { DefaultPalette } from 'office-ui-fabric-react/lib/Styling';
+
+export class MultiCountExample extends React.Component<{}, {}> {
+  constructor(props: ICardProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    const cardFrameContent = {
+      cardTitle: 'Multicount example',
+      cardDropDownOptions: [
+        {
+          key: 'Remove',
+          name: 'Remove',
+          icon: 'PageRemove',
+          ariaLabel: 'Remove card',
+          title: 'Remove card',
+          onClick: () => {
+            alert('Remove clicked');
+          }
+        }
+      ]
+    };
+
+    const rows = [
+      {
+        data: 13000,
+        bodyText: 'Flagged users',
+        annotaionText: 'Annotation Text',
+        color: DefaultPalette.accent,
+        type: AnnotationType.positive
+      },
+      {
+        data: 8000000,
+        bodyText: 'Risky sign-ins',
+        annotaionText: 'Decrease in the safety',
+        color: DefaultPalette.green,
+        type: AnnotationType.negative
+      },
+      {
+        data: 331100000,
+        bodyText: 'Risky sign-ins',
+        annotaionText: 'Annotation',
+        color: DefaultPalette.blue,
+        type: AnnotationType.neutral
+      }
+    ];
+
+    const contentAreaList = [
+      {
+        priority: Priority.Priority1,
+        cardContentType: CardContentType.MultiCount,
+        content: {
+          multiCountRows: rows
+        }
+      }
+    ];
+
+    const header = {
+      headerText: 'Header Text ',
+      annotationText: 'Annotation Text '
+    };
+
+    const actions: IAction[] = [
+      {
+        title: 'Action 1',
+        action: () => {
+          alert('Action1 clicked');
+        }
+      },
+      {
+        title: 'Action 2',
+        action: () => {
+          alert('Action2 clicked');
+        }
+      },
+      {
+        title: 'Action 3',
+        action: () => {
+          alert('Action3 clicked');
+        }
+      },
+      {
+        title: 'Action 4',
+        action: () => {
+          alert('Action4 clicked');
+        }
+      },
+      {
+        title: 'Action 5',
+        action: () => {
+          alert('Action5 clicked');
+        }
+      },
+      {
+        title: 'Action 6',
+        action: () => {
+          alert('Action6 clicked');
+        }
+      },
+      {
+        title: 'Action 7',
+        action: () => {
+          alert('Action7 clicked');
+        }
+      },
+      {
+        title: 'Action 8',
+        action: () => {
+          alert('Action8 clicked');
+        }
+      },
+      {
+        title: 'Action 9',
+        action: () => {
+          alert('Action9 clicked');
```

Comment:
nit: we have 9 action bar buttons in each card, leave all 9 of them in one remove the unnecessary 7 from others? Trying to clean up examples to show what they're intended to show

Reply:
removed action bar